core/tests/puppeteer-acceptance-tests/spec/exploration-editor-tests/create-exploration-and-change-basic-settings.spec.ts

```diff
@@ -38,12 +39,14 @@ ConsoleReporter.setConsoleErrorsToIgnore([
   /Error: Could not find the resource http:\/\/localhost:8181\/explorehandler\/features\/[a-zA-Z0-9]+\.?/,
   /Could not find the resource http:\/\/localhost:8181\/createhandler\/permissions\/[a-zA-Z0-9]+\.?/,
   /http:\/\/localhost:8181\/build\/webpack_bundles\/exploration_editor\.[a-f0-9]+\.bundle\.js/,
+  /http:\/\/localhost:8181\/create\/[a-zA-Z0-9]+#\/gui\/Introduction Failed to load resource: the server responded with a status of 404 \(Not Found\)/,
```

Comment:
If these errors are unexpected, these should be linked to issues, and we should drop the statement here when the issue is fixed.

In any case a comment should be left to explain the context of each error and the reason we exclude it.

Reply:
Actually I have already added a statement before all these errors.
These errors occurred when we deleted the exploration and then tried to access it. Most of these are `not found` type issues.

Reply:
Actually, I think that's a problem. Each console error should have its own comment (because otherwise, if we add new console errors and functionality in future, the comment won't apply to it). Secondly, only a single 404 console error should arise in cases like this, so it's probably worth filing a bug to fix that -- we shouldn't even get to the point of loading the rest of the page once the initial 404 is received.